core-services/openshift-priv/roles.yaml

```diff
@@ -0,0 +1,96 @@
+kind: List
+apiVersion: v1
+items:
+
+- kind: Namespace
+  metadata:
+    name: ocp-private
+
+- kind: ImageStream
+  metadata:
+    name: 4.1-priv
+    namespace: ocp-private
+  spec:
+    lookupPolicy:
+      local: true
```

Comment:
I think this will need to go in steps.

1) Add namespaces and RBACs (and also the RBACs that would allow the SA running applyconfig to create ImageStreams in this new namespace). These are admin resources.
2) In a followup PR, add imagestreams (and let `applyconfig` create it automatically).

We also need the _automatically tracks the content of the original ocp/4.X IS. This can be achieved by setting up these ImageStreams to have an import stanza and a short import period._ part of DPTP-820. You want to add that as a followup?

Reply:
I will remove the imagestreams from this PR and do a follow-up with DPTP-820.